bitnami/kong/templates/deployment.yaml

```diff
@@ -0,0 +1,268 @@
+          {{- end }}
+          {{- if .Values.kong.args }}
+          args: {{- include "kong.tplValue" (dict "value" .Values.kong.args "context" $) | nindent 12 }}
+          {{- end }}
+          lifecycle:
+            preStop:
+              exec:
+                command: ["/bin/sh", "-c", "kong quit"]
+          env:
+            {{- if .Values.service.exposeAdmin }}
+            - name: KONG_ADMIN_LISTEN_ADDRESS
+              value: "0.0.0.0"
+            {{- end }}
+            {{- if (eq .Values.database "postgresql") }}
+            - name: KONG_DATABASE
+              value: "postgres"
+            {{- if .Values.postgresql.usePasswordFile }}
+            - name: KONG_POSTGRESQL_PASSWORD_FILE
+              value: "/bitnami/kong/secrets/postgresql-password"
+            {{- else }}
+            - name: KONG_PG_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "kong.postgresql.secretName" . }}
+                  key: postgresql-password
+            {{- end }}
+            - name: KONG_PG_HOST
+              value: {{ include "kong.postgresql.host" . }}
+            - name: KONG_PG_USER
+              value: {{ include "kong.postgresql.user" . }}
+            {{- end }}
+            {{- if (eq .Values.database "cassandra") }}
+            - name: KONG_DATABASE
+              value: "cassandra"
+            {{- if .Values.cassandra.usePasswordFile }}
+            - name: KONG_CASSANDRA_PASSWORD_FILE
+              value: "/bitnami/kong/secrets/cassandra-password"
+            {{- else }}
+            - name: KONG_CASSANDRA_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "kong.cassandra.secretName" . }}
+                  key: cassandra-password
+            {{- end }}
+            - name: KONG_CASSANDRA_CONTACT_POINTS
+              value: {{ include "kong.cassandra.contactPoints" . }}
+            - name: KONG_CASSANDRA_PORT
+              value: {{ include "kong.cassandra.port" . | quote }}
+            - name: KONG_CASSANDRA_USER
+              value: {{ include "kong.cassandra.user" . | quote }}
+            {{- end }}
+            {{- if .Values.metrics.enabled }}
+            - name: KONG_NGINX_HTTP_INCLUDE
+              value: "/bitnami/kong/metrics-exporter/exporter.conf"
+            {{- end }}
+            {{- if .Values.kong.extraEnvVars }}
+            {{- include "kong.tplValue" (dict "value" .Values.kong.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          {{- if or .Values.kong.extraEnvVarsCM .Values.kong.extraEnvVarsSecret }}
+          envFrom:
+            {{- if .Values.kong.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ .Values.kong.extraEnvVarsCM }}
+            {{- end }}
+            {{- if .Values.kong.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ .Values.kong.extraEnvVarsSecret }}
+            {{- end }}
+            {{- end }}
```

Comment:
Indentation: This `{{- end }}` has two extra spaces

Reply:
Done. Also changed in the rest of the places